[2024-06-20T13:41:20.563+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: ELT-PROJECT.extract.extract_shippers scheduled__2024-06-19T00:00:00+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
